Library/Homebrew/dev-cmd/release-notes.rb

```diff
@@ -58,6 +58,7 @@ def release_notes
     end
 
     $stderr.puts "Release notes between #{previous_tag} and #{end_ref}:"
+    puts "Release notes for major and minor releases can be found in the [Homebrew blog](https://brew.sh/blog/)."
```

Comment:
- This is only used by maintainers (hence `dev-cmd`) so I think it's not worth adding this here (but can be argued around).
- This should also be on `$stderr` to allow it to be easily `| pbcopy` to generate the release notes
- If you're comparing to the last major or minor release, `--markdown` is used and it's been less than a month since the tag was created I think it's worth adding a `opoo` warning to point out it's been less than a month (and could add something to the release documentation too).

Reply:
> * This is only used by maintainers (hence `dev-cmd`) so I think it's not worth adding this here (but can be argued around).

The reason I'm adding this here is so that it will make it into the GitHub release notes. That way, running `brew release-notes | pbcopy` _does_ include it. I _do_ want this to be picked up by `pbcopy`.

> * This should also be on `$stderr` to allow it to be easily `| pbcopy` to generate the release notes

I want it to be picked up by `pbcopy` so I don't want it on `$stderr`

> * If you're comparing to the last major or minor release, `--markdown` is used and it's been less than a month since the tag was created I think it's worth adding a `opoo` warning to point out it's been less than a month (and could add something to the release documentation too).

Good idea. Should we always show the message, though, even when we can't determine what the next release number is going to be? I think you're the main one who uses this so I want it to work with your workflow. If you usually use `brew release-notes --markdown <old-ref> | pbcopy` there's no way for the command to tell whether the next release is a major/minor or patch release. If you also pass a `<new-ref>`, we might be able to tell.

But, since it's just a developer command and isn't run that often, we can also just display the warning every time.

Reply:
> The reason I'm adding this here is so that it will make it into the GitHub release notes. That way, running `brew release-notes | pbcopy` _does_ include it. I _do_ want this to be picked up by `pbcopy`.

Gotcha, fine with me then 👍🏻 

> Good idea. Should we always show the message, though, even when we can't determine what the next release number is going to be? I think you're the main one who uses this so I want it to work with your workflow. If you usually use `brew release-notes --markdown <old-ref> | pbcopy` there's no way for the command to tell whether the next release is a major/minor or patch release. If you also pass a `<new-ref>`, we might be able to tell.

`<old-ref>` is implicitly/by default if unspecified the last tag and the only time that's not useful is for major/minor releases. As a result, I think this can be scoped to just when it's passed through.

Reply:
So, just to clarify, here are the two options:

- `brew release-notes --markdown | pbcopy`: don't show any warning regardless of time fram
- `brew release-notes --markdown <old-ref> | pbcopy`: show warning if `<old-ref>` is less than one month ago

I'm fine with that as you're the one who tends to do this, but I feel like this is an arbitrary rule so I worry that it could change in the future and this functionality might be used unexpectedly.

Reply:
> * `brew release-notes --markdown | pbcopy`: don't show any warning regardless of time fram
> * `brew release-notes --markdown <old-ref> | pbcopy`: show warning if `<old-ref>` is less than one month ago

Yes, sounds good to me 👍🏻 



> I'm fine with that as you're the one who tends to do this, but I feel like this is an arbitrary rule so I worry that it could change in the future and this functionality might be used unexpectedly.

Could document the behaviour in the `--help` (and therefore also `man brew`) to clarify this if it puts you more at ease?